File: python/prime.py
import time
import sys

def run():
    limit = 10000
    if len(sys.argv) == 2: limit = int(sys.argv[1])

    start = time.clock()
    count = 0;
    number = 1
    while True:
        number += 1
        if not is_prime(number):
            continue
        count += 1
        if count == limit:
            print(str(limit) + "th prime is " + str(number))
            break
    print("[python] duration is " + str(time.clock() - start) + " sec")

def is_prime(number):
    for n in range(2, int(number/2)+1):
        if number % n == 0:
            return False
    return True

run()

File: php/prime.php
<?php
function isPrime($number)
{
	for($n = 2; $n <= (int)$number/2; $n++)
	{
		if($number % $n == 0) return false;
	}
	return true;
}

function main($items)
{
	$limit = 10000;
	if(count($items) == 2) $limit = intval($items[1]);

	$start = microtime(true);
	$list = array();
	for($number = 2; ; $number++)
	{
		if(!isPrime($number)) continue;
		$list[] = $number;
		if(count($list) == $limit)
		{
			print($limit . "th prime is " . $number . "\n");
			break;
		}
	}
	$time = microtime(true) - $start;
	printf("[php] duration is %2.2f sec\n", $time);
}

main($argv);
?>
